integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/authorization/CreateDhWithDatahubCreator.java

```diff
@@ -34,6 +34,11 @@
 import com.sequenceiq.it.cloudbreak.util.AuthorizationTestUtil;
 
 public class CreateDhWithDatahubCreator extends AbstractIntegrationTest {
+
+    private static Actor envcreatorb = Actor.useRealUmsUser(AuthUserKeys.ENV_CREATOR_B);
+
+    private static Actor zero = Actor.useRealUmsUser(AuthUserKeys.ZERO_RIGHTS);
```

Comment:
`zero` might be not that intuitive from the testContext. Can we change this to zeroRightsUser?
Also it looks strange to move these out to Class level. is this a pattern elsewhere?